extension-config.json

```diff
@@ -209,6 +209,33 @@
         "path": "/v1/shop/%(shopId)s/checkout_web_checkout_shopify?parsed=true",
         "key": "value.frontend.alias"
       }
+    },
+    "redisClientShopNumber": {
+      "type": "admin",
+      "destination": "backend",
+      "default": "",
+      "params": {
+        "label": "Shop number for Redis client configuration",
+        "type": "text"
+      }
+    },
+    "redisClientSecret": {
```

Comment:
we might need to make this a bigApi configuration to restrict its access like it is here https://github.com/shopgate/ext-shopify-cart/blob/master/extension-config.json#L29

Reply:
Made into configuration